### DISCUSSION TITLE
https://kelvinamoaba.com/blog/building-a-simple-smtp-server

### DISCUSSION THREAD

**Discussion by giscus** [AmoabaKelvin/blog#5] [Announcements]
# https://kelvinamoaba.com/blog/building-a-simple-smtp-server

 What does it mean to write your own smtp server Writing your own SMTP server basically means that you go by the SMTP RFCs and make sure that your system responds appropriately to whatever commands th

https://kelvinamoaba.com/blog/building-a-simple-smtp-server

**Comment by CoDedArch** [comment]
I haven't tried writing my own SMTP, I leverage on Django's in-built SMTP, Thanks Kelvin for sharing, I'll definitely give it a try.
